OS/Ywoosang/2025-10-31-Ywoosang-Linux-파일디스크립터.md

```
@@ -0,0 +1,141 @@
+자바에서 `Socket socket = new Socket(...)` 혹은 `ServerSocket.accept()`를 사용할 때 내부적으로 FD를 요청하여 TCP 연결을 관리한다.
+
+프로토콜 체계
+- AF_INET : IPv4
+- AF_INET6 : IPv6
+
+데이터 전송 방식
+- SOCK_STREAM : TCP 소켓
+- SOCK_DGRAM : UDP 소켓
+
+프로토콜 정보
+ - 앞선 인자만으로 프로토콜을 정확히 특정할 수 없는 경우에만 명시함. 특정 가능한 경우 위의 예시처럼 0으로 작성해도 무관
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)